src/components/Pages/Auth/shared/utils.js

```diff
@@ -6,6 +6,10 @@ export const AUTH_STATES = {
   USER_IS_AUTHENTICATED: "user_is_authenticated",
 };
 
+export const GUEST_USER_KEY ="GUEST_USER_EMAIL"
+export const emailIsInvalid =(email) => { 
+  return !email || !email.match(/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(.\w{2,3})+$/);
```

Comment:
I don't think this works with some of our test e-mail addresses (like brad+1@gmail.com)
The regex we use in the back-end is: 
        regex = '^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$'